### PR TITLE
Update Progenesis script FBMN

### PR DESCRIPTION
**Pull request by lfnothias**
Update the script
Update reference output files

### CODE REVIEW

**Comment by lfnothias** [comment]
@mwang87 it is ok to be merged.

**Comment by mwang87** [comment]
@lfnothias Can you make sure all the references are not to your repository, rather this repository?

**Comment by lfnothias** [comment]
@mwang87 Wait. This PR shouldn't have any absolute link. Is there actually some or are you refering to the documentation PR ?

File: feature-based-molecular-networking/README.md [hunk 1]
```diff
@@ -1,58 +1,85 @@
...
 
-Feature table should have three fixed columns
+Representative input files for each supported tools are available at:
+[https://github.com/CCMS-UCSD/GNPS_Workflows/tree/master/feature-based-molecular-networking/test/reference_input_file_for_formatter](https://github.com/CCMS-UCSD/GNPS_Workflows/tree/master/feature-based-molecular-networking/test/reference_input_file_for_formatter)
 
-1. Row ID
-2. Row m/z
-3. Row retention time
+Formatter script for each tool supported can be found in:
+[https://github.com/lfnothias/GNPS_Workflows/tree/master/feature-based-molecular-networking/tools/feature-based-molecular-networking/scripts](https://github.com/lfnothias/GNPS_Workflows/tree/master/feature-based-molecular-networking/tools/feature-based-molecular-networking/scripts)
```
Comment (mwang87) [inline feature-based-molecular-networking/README.md]: Here are the absolute links in the README

Comment (lfnothias) [reply feature-based-molecular-networking/README.md]: Thank you

**Comment by mwang87** [comment]
Can you also provide a bit more detail on the changes being made to the parser so we can note it in the change log?

**Comment by lfnothias** [comment]
- Fix for the parser with some metadata columns breaking the format.
- The parser now includes additional metadata columns in the output table
- Standardised the ion mobility column name in the output

**Pull request merged**